pyne/mesh.py

```diff
@@ -1510,6 +1511,102 @@ def _do_op(self, other, tags, op, in_place=True):
 
         return mesh_1
 
+
+class MeshTally(StatMesh):
+    """This class stores all information from all single mesh tally that
+    exists within some meshtal or state point file. Header information is
+    stored as attributes and the "mesh" attribute is a MOAB mesh with all
+    result and relative error data tagged. This class inherits from StatMesh,
+    exposing all statistical mesh manipulation methods.
```

Comment:
Overall I'd update this docstring to

```
... "mesh" attribute is a PyNE mesh object tagged with all result and relative error data. This class ....
```

Reply:
Updated in https://github.com/pyne/pyne/pull/1199/commits/e50076d1077c74b213305abdacbc9ceb5ff8e393.